docs/tutorials/code/chisel/rockcraft.yaml

```
@@ -1,17 +1,16 @@
-name: chisel-openssl
-summary: OpenSSL from Chisel slices
-description: A "bare" rock containing an OpenSSL installation created from Chisel slices.
+name: chisel-hello
```

```suggestion
name: chiselled-hello
```

(needs fixing in other places too)